.github/workflows/release.yaml

```diff
@@ -9,33 +11,50 @@ env:
 
 jobs:
   release-binaries:
-    name: Release ${{ matrix.arch }}-${{ matrix.os }}
-    runs-on: ubuntu-20.04
+    name: Release ${{ matrix.target }}
+    runs-on: ubuntu-22.04
     strategy:
       fail-fast: false
       matrix:
-        arch:
-          - aarch64
-          - x86_64
-        os:
-          - unknown-linux-gnu
-          - apple-darwin
+        include:
```

Comment:
I'd suggest to use dedicated runners on github actions over cross, e.g.:

```
    strategy:
      matrix:
        include:
          - { target: x86_64-unknown-linux-gnu, os: ubuntu-latest }
          - { target: x86_64-apple-darwin, os: macOS-latest }
          - { target: aarch64-apple-darwin, os: macOS-latest }
          - { target: x86_64-pc-windows-msvc, os: windows-latest }
```

Like that, we get to build natively on each platform.

Reply:
Building on both macos and linux simultaneously requires using cross (and having it fall through on macos) but this now works.